lib/export/coldp/files/type_material.rb

```diff
@@ -0,0 +1,153 @@
+# ID
+# nameID
+# citation
+# status
+# referenceID
+# locality
+# country
+# latitude
+# longitude
+# altitude
+# host
+# date
+# collector
+# institutionCode
+# catalogNumber
+# associatedSequences
+# sex
+# link
+# remarks
+#
+module Export::Coldp::Files::TypeMaterial
+
+  def self.locality(co)
+    [co.collecting_event&.cached_level0_geographic_name, co.collecting_event&.cached_level1_geographic_name, co.collecting_event&.cached_level2_geographic_name].compact.join(', ')
```

Comment:
There is a method to get all levels and their ranks IIRC.

Reply:
I haven't found a method, but I changed it to:

[co.dwc_country, co.dwc_state_province, co.dwc_county, co.dwc_locality].compact.join(', ')